documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -24,7 +24,7 @@ If a migration does not complete, you can create a new `Migration` CR, without c
 
 .Procedure
 
-. Obtain the SSL thumbprint of the vCenter host:
+. Obtain the SHA-1 fingerprint of the ESXi host:
```

Comment:
This is the fingerprint of the vCenter that we want, not ESXi !

Reply:
Fixed.